### ISSUE TITLE
Different switch for public channels while creating a channel.

### ISSUE THREAD

**Issue by Darshilp326** [RocketChat/Rocket.Chat#20488]
### Description:
While creating a channel there should be a different switch for the public channel as of now it's very confusing for new users while creating a public channel because the switch is showing red(OFF). 

### Steps to reproduce:

1. Create a new channel.

### Expected behavior:
Different switches for creating a private and public channel.

### Actual behavior:
Same switches are used which is very confusing.

### Server Setup Information:

- Version of Rocket.Chat Server: 3.11.0-develop
- Operating System: Ubuntu
- Deployment Method: 
- Number of Running Instances: 
- DB Replicaset Oplog: 
- NodeJS Version: 12.18.4
- MongoDB Version:4.2.8

### Client Setup Information

- Desktop App or Browser Version: Chrome
- Operating System: Ubuntu

### Additional context
Working!

**Comment by gabriellsh** [comment]
Hi, can you rephrase this issue so it doens't suggest a possible fix? The problem is not having only one toggle. Please, just describe it as being confusing.

There's no problem in suggesting a possible fix, but issues should describe problems first, if you wan't to comment a possible fix it should be done as a side note.

But thanks for the help! Sorry if this ~sounds~ reads a little harsh.

**Comment by Darshilp326** [comment]
@gabriellsh Hello, really sorry for the wrong title. Can you give me your suggestions so I  can work on this issue?

**Comment by aditya-mitra** [comment]
I guess if you can implement something like this would be very nice.

https://dribbble.com/shots/4898152-Switcher-PRD

**Comment by Darshilp326** [comment]
@aditya-mitra Hello! The issue is not with the switch but when we turn off that switch it shows a public channel which seems confusing if you want to make a public channel as the switch is in an OFF state.

**Comment by aditya-mitra** [comment]
> @aditya-mitra Hello! The issue is not with the switch but when we turn off that switch it shows a public channel which seems confusing if you want to make a public channel as the switch is in an OFF state.

I understood it. I mean to say if you could use ***such a switch*** which showed **public in off state** and **private in on state** (Similar to one I marked it in the dribbble shot) using the same colour for both. That would look very nice.

**Comment by gabriellsh** [comment]
I know some people are already working on this, and we even have a new design for it. It'll be fixed soon enough.
